docs/getting-started/installing-service-binding.md

```diff
@@ -14,3 +57,5 @@ _How can To be provided soon_
 
 _To be provided soon_
 
+[olm]: https://olm.operatorframework.io
```

Comment:
Those seems to be pre-requisites, not much about compatibility information.
Compatibility would:
- Kubernetes version: 1.18+ 
- Operator version: .... 
-